tests/resources/test-results.json

```diff
@@ -1,79 +1,138 @@
 [
```

Comment:
Please use the new mocks merged today into develop.

Reply:
rebased and fixed